configurations/tsconfig.json

```diff
@@ -23,9 +23,11 @@
         "@shopify/cli-kit": ["../packages/cli-kit/src/index.ts"],
         "@shopify/cli-kit/common/array": ["../packages/cli-kit/src/public/common/array.ts"],
         "@shopify/cli-kit/common/result": ["../packages/cli-kit/src/public/common/result.ts"],
-        "@shopify/cli-kit/node/ui": ["../packages/cli-kit/src/public/node/ui.tsx"],
-        "@shopify/cli-kit/node/presets": ["../packages/cli-kit/src/public/node/presets.ts"],
+        "@shopify/cli-kit/common/string": ["../packages/cli-kit/src/public/common/string.ts"],
```

Comment:
hey! the other day we noticed that if you add lines to this file but not to `package.json`, vscode will work fine, but the moment you try to import `@shopify/cli-kit/common/string` from another subpackage oclif will silently fail to load the module. is there a way to keep this file syncronized with the `package.json` exports? or at least add a test so we can catch this in CI?

Reply:
Opening a PR shortly

Reply:
https://github.com/Shopify/cli/pull/683/files